python/.gitignore

```diff
@@ -0,0 +1,2 @@
+.cache
```

Comment:
Where does `.cache` come from?

Reply:
I don't know actually, it just kind of appeared at some point. I'll remove this again for now, might not be necessary anymore

Reply:
Direnv, maybe.

Reply:
Ah no it's `clangd-tools`, maybe I'll add it again then